### PR TITLE
created desktop version styles for my portfolio

### PR DESCRIPTION
**Pull request by Ibranista**
#fixed wrapper alignment
#fixed alignment of work section

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Ibranista , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [x] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `description`
- [x] Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Naming your HTML file `index.html` as is the case by convention, instead of `home`
- Also providing some straightforward walkthrough on how to run the project locally in your README by providing some commands.
ex. `git clone <REPO LINK>`
`cd <REPO DIRECTORY>`
`...`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/home-desktop.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  --form-text: #979493;
+  --text-area: #fbf8f7;
+  --primary-family: "Crete Round";
+  --body-family: "Poppins";
+  --general-family: "Roboto";
+  --work-desc: "Inter";
+  --languages: "Meera Inimai";
+}
+
+/* navbar section */
```
[the remark this answers is not in the document]
Comment (Qoosim) [reply styles/home-desktop.css:26]: - [ ] You still need little work to make your mobile version look exactly like the figma design you chose. This is how it is still appeared on th mobile. 
your-mobile-headline

File: styles/home-desktop.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+/* main section */
+.wrapper {
+  box-sizing: border-box;
+  background: var(--background) url("../images/desktop-header.svg") no-repeat 100% 100%;
+  color: var(--white);
+  height: 100vh;
+  overflow: hidden;
+}
+
```
[the remark this answers is not in the document]
Comment (Qoosim) [reply styles/home-desktop.css:44]: - [ ] At screen size 768px, below is how your page appears which is different from how it appears on the figma design you chose.

### Your Design

your-work-title-sect


### Figma Design

temp-workstitle-sect

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Ibranista, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fherrerao — APPROVED** [review]
# ✅  🎆  STATUS APPROVED! ✅ 🎆 
#
Hi @Ibranista, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great Job Yes GIF by Hollywood Suite


## Highlights
✔️  No linter errors.
✔️  You used the correct GitHub-Flow.
✔️  Good README
✔️  Good Commit Messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**